.gitignore

```diff
@@ -46,6 +46,10 @@ Network Trash Folder
 Temporary Items
 .apdisk
 
+### Terraform ###
+**/.terragrunt-cache/*
+.terraform.lock.hcl
```

Comment:
Why did you add this file into the .gitignore. This is like a package.lock.json in nodejs.

Reply:
Because arm and intel binaries have different checksums and it doesn't make sense to use it here in multiplatform project, imho.

Reply:
[My bad](https://www.terraform.io/language/files/dependency-lock#new-provider-package-checksums), restored lockfiles